lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/MessagingTransportFactory.java

```diff
@@ -63,10 +64,10 @@ public MessagingTransportFactory() {
    * @param exHandler     a exception handler
    */
   @Override
-  public Transport create(final int port,
-                          final EventHandler<TransportEvent> clientHandler,
-                          final EventHandler<TransportEvent> serverHandler,
-                          final EventHandler<Exception> exHandler) {
+  public Transport getInstance(final int port,
```

Comment:
getInstance is usually used when it returns a singleton object.
If the method returns a new object every time it gets called, newInstance would be a better name.